geniza/corpus/management/commands/import_data.py

```diff
@@ -304,56 +366,27 @@ def import_documents(self):
 
         # create a reverse lookup for recto/verso labels used in the
         # spreadsheet to the codes used in the database
-        recto_verso_lookup = {
+        self.recto_verso_lookup = {
             label.lower(): code for code, label in TextBlock.RECTO_VERSO_CHOICES
         }
+        self.joins = []
+        self.docstats = defaultdict(int)
 
-        joins = []
-        docstats = defaultdict(int)
         for row in metadata:
-            if ";" in row.type:
-                logger.warning("skipping PGPID %s (demerge)" % row.pgpid)
-                docstats["skipped"] += 1
-                continue
+            self.import_document(row)
 
-            doctype = self.get_doctype(row.type)
-            fragment = self.get_fragment(row)
-            doc = Document.objects.create(
-                id=row.pgpid,
-                doctype=doctype,
-                description=row.description,
-            )
-            doc.tags.add(*[tag.strip() for tag in row.tags.split("#") if tag.strip()])
-            # associate fragment via text block
-            TextBlock.objects.create(
-                document=doc,
-                fragment=fragment,
-                # convert recto/verso value to code
-                side=recto_verso_lookup.get(row.recto_verso, ""),
-                region=row.text_block,
-                subfragment=row.multifragment,
-            )
-            self.add_document_language(doc, row)
-            docstats["documents"] += 1
-            # create log entries as we go
-            self.log_edit_history(
-                doc, self.get_edit_history(row.input_by, row.date_entered, row.pgpid)
-            )
-            # parse editor & translator information to create sources
-            # and associate with footnotes
-            editor = row.editor.strip(".")
-            if editor and editor not in self.editor_ignore:
-                self.parse_editor(doc, editor)
-            # treat translator like editor, but set translation flag
-            if row.translator:
-                self.parse_editor(doc, row.translator, translation=True)
-
-            # keep track of any joins to handle on a second pass
-            if row.joins.strip():
-                joins.append((doc, row.joins.strip()))
+        # update id sequence based on highest imported pgpid
+        self.update_document_id_sequence()
+
+        for row in demerged_metadata:
+            # overwrite document if it already exists
+            if row.pgpid and Document.objects.filter(id=row.pgpid).exists():
```

Comment:
Did this happen at all in your testing?

Reply:
yes, should we give a heads up to the pgp team?
```
WARNING:import:Overwriting PGPID with demerge 31511
```

Reply:
I just checked the spreadsheet — that one has two types separated by a semicolon, so I think we _should_ be skipping it. 

What happens if you don't check and you try to save a record for a pgpid that already exists? Does it error?

Reply:
The PGPID was manually defined twice in the demerge spreadsheet. That's what was causing the error. It was correctly skipped in the metadata spreadsheet.

<img width="603" alt="Screen Shot 2021-05-17 at 4 46 06 PM" src="https://user-images.githubusercontent.com/26414213/118554402-632a6200-b72f-11eb-9563-244406443f75.png">

Reply:
ah, ok — great that you're checking, then! Thanks for finding and correcting it